Source/EasyNetQ/ConnectionConfiguration.cs

```diff
@@ -90,21 +92,29 @@ private void SetDefaultClientProperties(IDictionary<string, object> clientProper
             var hostname = Environment.MachineName;
             var product = Product ?? applicationName;
             var platform = Platform ?? hostname;
+            var name = Name ?? applicationName;
```

Comment:
@zidad sorry for delay. I have tried to test with .net core client and version EasyNetQ 2.0.4-unstable0010 - however it looks like there are no changes there.
Maybe you can specify which exactly nuget version?